cloud-run/filtering/requirements.txt

```diff
@@ -0,0 +1,10 @@
+google-cloud-functions
+google-cloud-logging
+pittgoogle-client>=0.3.12
+
+# for Cloud Run
+# https://cloud.google.com/run/docs/quickstarts/build-and-deploy/deploy-python-service
+# pinned following quickstart example. [TODO] consider un-pinning
+Flask==3.0.3
+gunicorn==23.0.0
+Werkzeug==3.0.3
```

Comment:
I think I'm in favor of unpinning these here, at least just to see what happens.